src/ol/render/canvas/Immediate.js

```diff
@@ -946,6 +946,8 @@ class CanvasImmediateRenderer extends VectorContext {
     const textAlign = textState.textAlign
       ? textState.textAlign
       : defaultTextAlign;
+    // Ignore `textState.justify` here,
+    // since rich-text is not supported for the immediate rendering API.
```

Comment:
Maybe replace with
```suggestion
    //FIXME Implement `justify`
```
or remove the comment.

Reply:
Thx. I'll remove the comment, because, as far as I know, having multi-line text for immediate rendering is not possible yet.